bundles/org.openhab.binding.pentair/README.md

```diff
@@ -72,133 +87,200 @@ The following table shows the available configuration parameters for each thing.
-| mode                 | Number    | Pump mode (readonly)                                         |
-| rpm                  | Number    | Pump RPM (readonly)                                          |
-| power                | Number    | Pump power in Watts (readonly)                               |
-| error                | Number    | Pump error (readonly)                                        |
-| ppc                  | Number    | Pump PPC? (readonly)                                         |
-
-## Full Example
-
-The following is an example of an item file (pentair.items), you can change the °F to °C if you are using metric temperature units:
+## Things & Channels
+
+### Thing: Controller
+
+Represents and interfaces with a Pentair pool controller in the system.  This binding should work for both Intellitouch and EasyTouch systems.
+Feature availability is dependent on the version of hardware and firmware versions of your specific controller.
+
+#### Synchronize Time
+
+This configuration setting will instruct the binding to automatically update the controller's clock every 24 hours with the value from the openHab server.
+This is useful to keep the pool system clock set correct and automatically adjust for daylight savings time.
+
+| Channel Group                     | Channel   | Type   |    | Description                                              |
+| :------------------------------:  | :-------: | :----: | :-: | :------------------------------------------------------- |
+| pool, spa, aux[1-8], feature[1-8] | switch    | Switch | RW | Indicates the particulcar circuit or feature is on or off.  | 
+| "                                 | name      | String | R  | Name of circuit |
+| "                                 | feature   | String | R  | Feature of ciruit |
+| poolheat, spaheat                 | setpoint  | Number:Temperature | RW | Temperature setpoint |
+| "                                 | temperature | Number:Temperature | R | Current water temperature.  Note, the temperature is only valid while in either pool or spa mode. |
+| "                                 | heatmode  | String | R  | Heat mode configured.  Values: NONE, HEATER, SOLARPREFERRED, SOLAR |
+| schedule[1-9]                     | schedule  | String | RW | Summary string of schedule.  |
+| "                                 | type      | String | RW | Type of schedule.  Note, to actually write the program to the controller, this channel must be written to with the same value 2 times within 5s. Values: NONE, NORMAL, EGGTIMER, ONCE ONLY |
+| "                                 | start     | Number | RW | Time of day to start schedule expressed in minutes.  |
+| "                                 | end       | Number | RW | Time of day to end schedule expressed in minutes. In the case of EGG TIMER, this shoud be the duration. |
+| "                                 | circuit   | Number | RW | Circuit/Feature the schedule will control. |
```

Comment:
What's a circuit in this case? Is the feature's number refering to the channel group? If so, you could add a note here.